ksql-engine/src/test/java/io/confluent/ksql/testutils/EmbeddedSingleNodeKafkaCluster.java

```diff
@@ -163,4 +187,66 @@ public void createTopic(String topic,
     broker.createTopic(topic, partitions, replication, topicConfig);
   }
 
+  public static Builder newBuilder() {
+    return new Builder();
+  }
+
+  private Properties effectiveBrokerConfigFrom() {
+    Properties effectiveConfig = new Properties();
+    effectiveConfig.put(KafkaConfig.ListenersProp(), "PLAINTEXT://:0");
+    effectiveConfig.putAll(brokerConfig);
+    effectiveConfig.put(KafkaConfig.ZkConnectProp(), zookeeper.connectString());
+    effectiveConfig.put(KafkaConfig.DeleteTopicEnableProp(), true);
+    effectiveConfig.put(KafkaConfig.LogCleanerDedupeBufferSizeProp(), 2 * 1024 * 1024L);
+    effectiveConfig.put(KafkaConfig.OffsetsTopicReplicationFactorProp(), (short) 1);
+    return effectiveConfig;
+  }
+
+  private static void createServerJaasConfig() {
+    try {
+      final String jaasConfigContent = createJaasConfigContent();
+      final File jaasConfig = File.createTempFile("jaas_conf", null);
+      jaasConfig.deleteOnExit();
+      Files.write(jaasConfigContent, jaasConfig, StandardCharsets.UTF_8);
+
+      System.setProperty(JaasUtils.JAVA_LOGIN_CONFIG_PARAM, jaasConfig.getAbsolutePath());
```

Comment:
are we unsetting these somewhere?

Reply:
They are per-jvm so unsetting wouldn't have any affect.